File: f044(slime)/Python/slime.py
king_slime, small_slime = map(int, input().split())
small_slime /= king_slime
king_slime = 1
total = king_slime + small_slime
days = 0
while  total != 1:
    total /= 2
    days += 1
print(days)

File: f044(slime)/C++/slime.cpp
#include <iostream>
#include <cmath>

int main(void) {

  int kingSlime, smallSlime;
  std::cin >> kingSlime >> smallSlime;

  int total = 1 + smallSlime/kingSlime;
  
  // int days = 0;
  // while(total != 1) {
  //   total /= 2;
  //   days++;
  // }
  // std::cout << days << '\n';

  std::cout << log2(total) << '\n';

  return 0;
}
